README.md

```diff
@@ -164,24 +190,26 @@ Pros of creating a new tool:
   It would be hard to do that while preserving compatibility in jpm.
 * Creating SDK-based add-ons was overly complicated. With web extensions you no
   longer need to convert your source into legacy artifacts and you won't need
-  boostrapping scripts.
+  bootstrapping scripts.
 * There are superior features in Firefox now for developing extensions such
   as [loading](https://blog.mozilla.org/addons/2015/12/23/loading-temporary-add-ons/)
-  from source code instead of a packaged XPI. It will be
-  easier to reimagine a new tool around these work flows rather than
-  adjust jpm's existing work flows.
+  from source code instead of a packaged XPI. It will be easier to reimagine a new tool around these workflows rather than adjusting jpm's existing workflows.
 * jpm's functional tests are slow, brittle and hard to run. There are flaky
   time-outs and we've run out of *low hanging fruit* fixes at this point.
-* Most of jpm's code was not designed to be unit testable which makes it hard to
+* Most of the jpm's code was not designed to be unit testable which makes it hard to
```

Comment:
Since `jpm` is used here as a proper noun it sounds weird prefixed with an article